### ISSUE TITLE
Multiple comments in LESS file is failing (0.3.9)

### ISSUE THREAD

**Issue by cognivator** [SC5/sc5-styleguide#560]
The following source causes a console message, "TypeError: Cannot call method 'substr' of undefined" right after the normal generator message, "Generating style guide to output dir: styleguide"

NOTE: This is a greatly simplified version of my original file, which generated fine in 0.3.8.
```
// Colors
// 
// Our colors
//
// Styleguide 1.0

// Pallette
//
// Our palette
//
// Styleguide 1.1
```

Either KSS comment on its own generates correctly.

**Comment by hannu** [comment]
I'm unable to reproduce this issues with the example file you provided. For me the style guide is generated correctly.

**Comment by hannu** [comment]
Due to vast amount of issues with the newer gonzales, sc5-styleguide 0.3.10 is now released containing older gonzales (3.0.0-12).

Please re-open this issue if this issue still exists with the 0.3.10
